spec/octopus/serverspec/json/api/spaces/all.json

```diff
@@ -27,7 +27,7 @@
   {
     "Id": "Spaces-2",
     "Name": "Octopus",
```

Comment:
Consider changing this name - it makes the tests less clear

Reply:
Affects a few other tests by the look of things. But worth doing.